### ISSUE TITLE
Missing unit test for routine add_subtitle_item.MICRODVD_SUBTITLE

### ISSUE THREAD

**Issue by LfLanzoni** [nmaguirre/eiffel-subtitle-converter#36]

**Comment by LfLanzoni** [comment]
Soved 4642744
 https://github.com/nmaguirre/eiffel-subtitle-converter/commit/4642744afa8d775d65e11ae8a8d682e69754b57a

**Comment by LfLanzoni** [comment]
soved acd54960cfbd16a99ce7f205e21121abf4d070df

**Comment by saiema** [comment]
test_add_subtitle_item_valid doesn't test anything

```
create sub.make
sub.add_subtitle_item(20,100,"text1")
sub.add_subtitle_item(201,300,"text3")
sub.add_subtitle_item(101,200,"text2")
res := true
assert ("Add Subtitle is ok",res)
```

The assert should check that the three items are present in sub.items in the correct order with the correct values, that is, [(20,100,"text1"), (101,200,"text2"), (201,300,"text3")]

test_add_subtitle_item_invalid doesn't test routine add_subtitle_item, a correct negative test should be something like

```
sub.add_subtitle_item(20,100,"text1")
sub.add_subtitle_item(80,101,"text3")
```

**Comment by LfLanzoni** [comment]
Sorry soved 84954154f8399210e0acdd0a2541dd8f58c30b29

**Comment by nricci** [comment]
Well Done! Thanks!

**Comment by gamassimino** [comment]
sorry but the test fails because you should use the feature "is equal" instead of the operator "=" in the assert

**Comment by LfLanzoni** [comment]
Sorry, solved 056d4015868a688119d6b628747666b520f2b2b5
